card/mod/core/set/all/name.rb

```diff
@@ -55,21 +55,25 @@ def key= newkey
 def update_subcard_names new_name, name_to_replace=nil
   return unless @subcards
   subcards.each do |subcard|
```

Comment:
this is mostly cleanup. the change is _not_ altering the subcard name if the subcard is real.